keymanager-client/src/client.rs

```diff
@@ -149,6 +132,21 @@ impl RemoteClient {
             keys_cache_sizes,
         )
     }
+
+    /// Sets client allowed enclaves from key manager policy.
+    pub fn set_policy(&self, signed_policy_raw: Vec<u8>) -> () {
+        let untrusted_policy: SignedPolicySGX = match cbor::from_slice(&signed_policy_raw) {
+            Ok(sp) => sp,
+            Err(err) => panic!("error obtaining list of KM enclaves: {}", err),
```

Comment:
Should this method maybe just return an error instead of panicking? And then the caller can decide to panic. E.g. just make this method return `Fallible<()>` and then use `cbor::from_slice(&signed_policy_raw)?`.